fe/fe-core/src/main/java/com/starrocks/external/iceberg/IcebergUtil.java

```diff
@@ -57,9 +57,9 @@ public static IcebergCatalog getIcebergCatalog(IcebergTable table)
     /**
      * Returns the corresponding hive catalog implementation.
      */
-    public static IcebergCatalog getIcebergHiveCatalog(String metastoreUris)
+    public static IcebergCatalog getIcebergHiveCatalog(String metastoreUris, Map<String, String> icebergProperties)
```

Comment:
This api has been changed, so the unit test will exit with failure it break the mock semantics below in IcebergTableTest
```
new MockUp<IcebergUtil>() {
            @Mock
            public IcebergCatalog getIcebergHiveCatalog(String uris) {
                return icebergCatalog;
            }
        };
```